core/templates/dev/head/components/SearchBarDirective.js

```diff
@@ -133,7 +149,7 @@ oppia.directive('searchBar', [function() {
           if (evt.keyCode == 13 || evt.keyCode == 32) {
             _onSearchQueryChangeExec();
           } else {
-            oppiaDebouncer.debounce(_onSearchQueryChangeExec, 400)();
+            oppiaDebouncer.debounce(_onSearchQueryChangeExec(), 400)();
```

Comment:
This still looks wrong. There shouldn't be a (), since you're passing in a function.